tests/cpptests/micro-benchmarks/benchmark_idlist_iterator.cpp

```diff
@@ -58,7 +58,9 @@ class BM_IdListIterator : public benchmark::Fixture {
 template <typename IteratorType>
 bool BM_IdListIterator<IteratorType>::initialized = false;
 
-#define DOCIDS_SCENARIOS() Arg(10)->Arg(100)->Arg(1000)->Arg(100000)
+#define READ_DOCIDS_SCENARIOS() Arg(100)
+#define SKIPTO_DOCIDS_SCENARIOS() Arg(100)->Arg(1000)
+
```

Comment:
I think there was a misunderstanding
1. Let's remove these definitions completely
2. Set `numDocuments` in `Setup` to a high number so the benchmark has enough data to test